content/en/agent/logs/advanced_log_collection.md

```diff
@@ -443,6 +443,92 @@ More examples:
 | 2020-10-27 05:10:49.657  | `\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}\.\d{3}` |
 | {"date": "2018-01-02"    | `\{"date": "\d{4}-\d{2}-\d{2}`                |
 
+### Automatic Multi-line aggregation
+With agent 7.32+ `auto_multi_line_detection` can be enabled which allows the agent to detect [common multi-line patterns](https://github.com/DataDog/datadog-agent/blob/main/pkg/logs/decoder/auto_multiline_handler.go#L195) automatically. This feature is currently in beta. 
+
+`auto_multi_line_detection` can be enabled globally in the `datadog.yaml`
+
+```yaml
+logs_config:
+  auto_multi_line_detection: true
+```
+
+It can also be enabled or disabled (overriding the global config) per log configuration:
+
+{{< tabs >}}
+{{% tab "Configuration file" %}}
+
+```yaml
+logs:
+  - type: file
+    path: /my/test/file.log
+    service: cardpayment
```

Comment:
nit: let's use a different service name, this one seems to refer to the credit card scrubbing logic :)

(same for the kubernetes example)

Reply:
changed to `testApp`